src/bt/Node.cpp

```diff
@@ -79,7 +80,10 @@ void Node::NodeTerminate(Status s) {
 
 unsigned long long Node::getAmountOfTicks() const { return amountOfTicks; }
 
-void Node::giveProperty(std::string a, std::string b) { std::cerr << "giveProperty in Node.cpp, should never be called" << std::endl; }
+void Node::giveProperty(std::string a, std::string b) {
+    rtt_error("giveProperty in Node.cpp, should never be called");
+    assert(false);
```

Comment:
Would this not crash our entire code?

Reply:
No it should not but, but I know a cleaner solution: making this purely virtual.